proto/planetmintgo/machine/machine.proto

```diff
@@ -13,8 +13,9 @@ message Machine {
   uint64 precision = 6; 
   string issuerPlanetmint = 7; 
   string issuerLiquid = 8; 
-  string machineId = 9; 
+  string machineId = 9;
   Metadata metadata = 10; 
+  uint32 type = 11;
```

Comment:
Question: you define type as uint32, but in openapi.yml it is `int64`. I understand that a uint32 fits into an int64, but can this lead to unwanted behaviour?